Source/WebCore/page/scrolling/ScrollingTree.h

```diff
@@ -332,6 +335,7 @@ class ScrollingTree : public ThreadSafeRefCounted<ScrollingTree> {
     bool m_isHandlingProgrammaticScroll { false };
     bool m_isMonitoringWheelEvents { false };
     bool m_scrollingPerformanceTestingEnabled { false };
+    bool m_overlayScrollbarsEnabled { false };
```

Comment:
Not now, but maybe in future we should group all these settings into an OptionSe<> and use one changed flag for all of them.